frontend/src/app/shared/components/charts/charts.component.ts

```diff
@@ -73,6 +73,10 @@ export class ChartsComponent extends BaseComponent implements OnInit, AfterViewI
             colorScheme: chartOptions?.colorScheme ?? {
                 domain: ['#5AA454', '#A10A28', '#C7B42C', '#AAAAAA']
             },
+            colorSchemeHeatMap: chartOptions?.colorScheme ?? {
+                domain: ['#146738', ' #2C9653', '#6CBA67', ' #A9D770',
+                    '#DBED91', '#FDDE90', '#FAAD67', '#EF6D49', '#D3342D', '#A10A28']
+            },
```

Comment:
Don't create your own property, just specify the property in the `heat-map.component.ts` in `initChartSettings()`

Reply:
Fix